tools/sistr_cmd/sistr_cmd.xml

```diff
@@ -161,72 +164,45 @@
   </outputs>
   <tests>
     <test>
-      <param name="input_fastas" value="AE014613-699860.fasta"/>
+      <param name="input_fastas" value="AE014613-699860.fasta" ftype="fasta"/>
       <param name="output_format" value="tab"/>
       <output 
         name="cgmlst_profiles" 
         value="cgmlst-profiles.csv"
         ftype="csv"
-        lines_diff="2">
-        <assert_contents>
-          <has_text text=",,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,3969539340,2545200385,225275747,2955003506,2353669245,2666669453,1672513023,3779563470,1301843222,2161147266,607954140,3680021500,2914087704,1062106200,3673111880,1314942441,1367997025,3293595301,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,1528212814,2110459436,4160823845,1648892875,2084418558,1638162324,469721942,1317894045,1973458150,926214622,2197498164,398274060,,,,,,,,,,1154766063,278162969,490843778,3950769715,,,,,,4203409135,2061008354,,,,,,,1052128508,,,1510445340,,,4065472468,,,,,,1495737522,,,,,,,,3076491138,712233770,3105746335,625241463,3016847250,1928860657,2229984332,1341416065,2978539204,1175502179,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,1328452594,2372254687,2640609716,3051501604,3258707132,,,,,,,,,,,,1007978530,,3646345463,1109678443,,,,,,,,,"/>
-        </assert_contents>
+        compare="re_match">
```

Comment:
I think you can remove `re_match` here and just leave the default comparison as `diff` since there's no regular expressions we are comparing against.

Reply:
Since the file contents should not change, the `diff` should work. I was just afraid of end of the line character might change (`\n` or `\r`) and though that `re_match` could work more agnostically but I never used this option before

Reply:
Thanks. Yeah, end of line characters can sometimes cause issues. But I don't think it will cause issues in this case. I've actually never used `re_match` either.